functests/utils/nodes/nodes.go

```diff
@@ -278,12 +278,15 @@ func MatchingOptionalSelectorPTP(toFilter []ptpv1.NodePtpDevice) ([]ptpv1.NodePt
 	return res, nil
 }
 
-// ValidateNodeIsAvailableForSelector check if any nodes are available for given nodeSelector
-func ValidateNodeIsAvailableForSelector(nodeSelector map[string]string) bool {
+// NodesAvailableForSelector returns nodes available for given nodeSelector
```

Comment:
The package name is already "nodes", you can just call this "AvailableForSelector"

Reply:
Done